lms/envs/common.py

```diff
@@ -1207,6 +1207,7 @@ def _make_mako_template_dirs(settings):
     COMMON_ROOT / "static",
     PROJECT_ROOT / "static",
     NODE_MODULES_ROOT / "@edx",
+    "/edx/src/adg-edx-theme/edx-platform/adg/lms/static",
```

Comment:
Is it intentional? If Yes, Why are we adding this path here?

Reply:
Yes it is intentional. Actually it wasn't working for the js files that I put in the theme directory. Worked when I updated static files path.